README.md

```diff
@@ -1,7 +1,7 @@
 ## To-do-list ![To-do-list](https://img.shields.io/badge/Microverse-blueviolet)
 
 
-To do list is a basic website, On page load render the dynamically created list of tasks in the dedicated placeholder. The list should appear in order of the index values for each task also implemented the CRUD (create, update, delete) methods.
+To do list is a task mangement webapp, On page load render the dynamically created list of tasks in the dedicated placeholder. The list should appear in order of the index values for each task also implemented the CRUD (create, update, delete) methods.The user will also be able to mark task completion clicking on checkbox and undo it.
 
```

Comment:
- [ ] Please use the `change` event listener on the checkbox as per the requirement, actually, you have used the click event listener, kindly change that
<img width="399" alt="chk" src="https://user-images.githubusercontent.com/79058364/159920556-28ea5dc6-98ce-4d69-a105-36f1c3f7c701.png">

- [ ] Consider using the `filter` array method to filter all completed items on the `removeAllcompleted function which is in your case this below ⬇️ 
<img width="464" alt="filter" src="https://user-images.githubusercontent.com/79058364/159920990-32a6193f-bf1e-4de6-809f-e31304b308ac.png">